stdlib/public/runtime/ProtocolConformance.cpp

```diff
@@ -771,8 +771,8 @@ static void validateDyldResults(
   };
 
   if (dyldCachedConformanceDescriptor) {
-    if (std::find(conformances.begin(), conformances.end(),
-                  dyldCachedConformanceDescriptor) == conformances.end()) {
+    if (conformances.find(dyldCachedConformanceDescriptor) ==
```

Comment:
`conformances` is `llvm::SmallVector` it doesn't have `find` method but even if it did it couldn't be `O(log n)` because there is no guarantee that the vector is sorted...